thrusters_blue_robotics_t500.orogen

```diff
@@ -18,14 +18,19 @@ task_context "Task" do
     # commands (a float) and the second their respective pwm duty cyle (an int)
     property "command_to_pwm_table_file_path", "/std/string"
 
+    # A vector indicating wether the helices of the joints are inverted
+    # 1 for not inverted CCW helice
+    # -1 for inverted CW helice
+    property "helices_alignment", "/std/vector</int>"
```

Comment:
I would prefer that you use a THRUSTER_HELICE_CW and THRUSTER_HELICE_CCW enum on the interface. This makes it a lot clearer which is which (no need to know that CW is positive and CCW negative)

Reply:
Note that this is C++ ... you can make the value of ..._CW 1 and _CCW -1 to simplify the conversion to int in the configureHook